NitroxServer-Subnautica/GameLogic/Prefab.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxServer.UnityStubs;
+
+namespace NitroxServer.GameLogic
+{
+    public class Prefab
```

Comment:
Can we use `UwePrefab`?

Reply:
I think i did switch over to UwePrefab due to the merging